src/main/java/seedu/duke/Duke.java

```diff
@@ -21,6 +20,9 @@
     private Timetable timetable;
     private TagManager tagManager;
 
+    private static final String WELCOME_MSG_STRING = "Hi";
```

Comment:
Think maybe we can have this at the very beginning?
```suggestion
    private static final String WELCOME_MSG_STRING = "Welcome to NotUS! Type \"help\" if you need to see a list of commands and their usages.";
```